omero/new/index.html

```diff
@@ -27,21 +33,15 @@ <h2 class="header-blue">NEW Features</h2>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/viewing-data.html#screen" target="_blank">View user guide</a>
             </div>
             <div class="omero-feature-container column text-center">
-                <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_lookup-table.svg">
-                <h4>LookUp Table support</h4>
-                <p>Select a LookUp Table for your data in the UI.</p>
-                <a class="button hollow tiny" href="http://help.openmicroscopy.org/viewing-data.html#rendering" target="_blank">View user guide</a>
-            </div>
-            <div class="omero-feature-container column text-center">
                 <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_plugin.svg">
                 <h4>Open With...</h4>
                 <p>Launch custom plugins via OMERO.web.</p>
                 <a class="button hollow tiny" href="http://help.openmicroscopy.org/web-client.html" target="_blank">View user guide</a>
             </div>
             <div class="omero-feature-container column text-center">
                 <img class="thumbnail omero-feature" alt="OMERO feature" src="{{ site.baseurl }}/img/icons/omero-features-icons_reverse-intensity.svg">
-                <h4>Reverse intensity rendering</h4>
-                <p>Reverse or invert image intensity.</p>
+                <h4>Invert intensity rendering</h4>
+                <p>Invert the display of image colors according to intensity.</p>
```

Comment:
I find this phrasing a little confusing.  We're not "inverting the display", we are inverting the intensity of a channel for display.